content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -1231,5 +1231,5 @@ it is created.
 * `Deployment` is a top-level resource in the Kubernetes REST API.
   Read the {{< api-reference page="workload-resources/deployment-v1" >}}
   object definition to understand the API for deployments.
-* Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
-  you can use it to manage application availability during disruptions.
+* Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how you can use it to manage application availability during disruptions.
```

Comment:
Why don't you wrap the long line as we do for the bullet on line 1231?

Reply:
Hi @tengqm,
Applied your suggestions, could you please take a look into it once!!

Reply:
It's still not wrapped, try to line break at ~80 characters similar to the previous version.

Reply:
@shannonxtreme, @tengqm,
Hope i fixed the issue now, could you PTAL!!

Deploy preview: https://deploy-preview-38230--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/controllers/deployment/#what-s-next